.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+## Related Issue
+<!-- If this PR addresses an issue, link it here (e.g., "Closes #123") -->
+
+## Description
+<!-- A description of the changes made in this PR -->
+
+## Screenshots (if applicable)
+<!-- Attach any screenshots that help explain your changes -->
```

Comment:
I would move this section to the end of the list, since screenshots take a lot of real estate, pushing the next two points at the far bottom

Reply:
Updated